example_gh_test.go

```diff
@@ -164,31 +164,24 @@ func ExampleGQLClient_Mutate_simple() {
 		AddStar struct {
 			Starrable struct {
 				Repository struct {
-					Stargazers struct {
-						Nodes []struct {
-							Login string
-						}
-					} `graphql:"stargazers(first: $first)"`
+					StargazerCount int
 				} `graphql:"... on Repository"`
 				Gist struct {
 					StargazerCount int
-					Url            string
 				} `graphql:"... on Gist"`
 			}
 		} `graphql:"addStar(input: $input)"`
 	}
 	variables := map[string]interface{}{
 		"input": githubv4.AddStarInput{
-			ClientMutationID: githubv4.NewString("addStar"),
-			StarrableID:      "R_kgDOF_MgQQ",
+			StarrableID: githubv4.NewString("R_kgDOF_MgQQ"),
```

Comment:
Nit: for corectness, should this be `githubv4.NewID` instead of NewString? I don't think it makes a difference at runtime